Chinese/海贼王/海贼王.md

```diff
@@ -0,0 +1 @@
+海贼王非官方电台是国内最大的由海迷为海迷无私奉献的谈话电台节目，主打《海贼王》剧情回顾及未来剧情预测，辅以海贼百科、海贼日语和海贼恶搞新闻等相关节目。目前状态每月更新三到四次，平均每周一次。主要更新地点为荔枝fm、喜马拉雅fm、网易云音乐。在Podcasts也同步更新。
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.